gulpfile.js

```diff
@@ -45,18 +48,57 @@ gulp.task('format', () => {
 });
 
 gulp.task('tsc', function(done) {
-  runSpawn(done, process.execPath, ['node_modules/typescript/bin/tsc']);
+  runSpawn(process.execPath, ['node_modules/typescript/bin/tsc'], done);
 });
 
 gulp.task('prepublish', function(done) {
   runSequence('tsc', 'copy', done);
 });
 
-gulp.task('default',['prepublish']);
-gulp.task('build',['prepublish']);
+gulp.task('default', ['prepublish']);
+gulp.task('build', ['prepublish']);
 
-gulp.task('test', ['format', 'build'], function(done) {
-  var opt_arg = [];
-  opt_arg.push('node_modules/jasmine/bin/jasmine.js');
-  runSpawn(done, process.execPath, opt_arg);
+// Command line commands
+gulp.task('update', ['build'], function(done) {
+  runSpawn(process.execPath, ['bin/webdriver-manager', 'update', '--android',
+      '--android-accept-licenses'], done);
 });
+gulp.task('start', ['build', 'shutdown'], function(done) {
```

Comment:
fyi, Craig and I have talked about moving away from gulp to just shell scripts. That's similar to what Angular does for their CI. I think it's a bit easier to understand, when all you're doing is spawning a bunch of processes in sequence.